crates/rari-types/src/locale.rs

```diff
@@ -112,6 +112,24 @@ static LOCALES_FOR_GENERICS_AND_SPAS: LazyLock<Vec<Locale>> = LazyLock::new(|| {
         .collect::<Vec<_>>()
 });
 
+static TRANSLATED_LOCALES: LazyLock<Vec<Locale>> = LazyLock::new(|| {
+    let translated_locales = [
+        Locale::Es,
+        Locale::Fr,
+        Locale::Ja,
+        Locale::Ko,
+        Locale::PtBr,
+        Locale::Ru,
+        Locale::ZhCn,
+        Locale::ZhTw,
+    ];
```

Comment:
```suggestion
    let translated_locales = [
        Locale::De,
        Locale::Es,
        Locale::Fr,
        Locale::Ja,
        Locale::Ko,
        Locale::PtBr,
        Locale::Ru,
        Locale::ZhCn,
        Locale::ZhTw,
    ];
```

Reply:
Agreed, I was not aware that `de` is now officially supported